manifest.json

```diff
@@ -75,30 +83,33 @@
         "mssql"
       ],
       "default": "postgres",
-      "required" : true
+      "required": true
     },
     {
       "name": "db_host",
       "type": "string",
       "title": "Host Name",
+      "required": true,
       "description": "The public host name of your database server."
     },
     {
       "name": "db_port",
-      "type": "string",
+      "type": "number",
```

Comment:
Make sure we know what happens when it's a string, then manifest gets updated.

try with bad values too, like a string that isn't a number.

Reply:
Migrating from a string to a number conserves the original port number if this number as a string was valid. If not, the field is reseted.